### PR TITLE
Update generator Dockerfile

### PR DESCRIPTION
**Pull request by SuperQ**
* Use a single RUN to build as a single layer.
* Bake MIBs into the image.
* Setup MIBDIRS ENV.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by daenney** [comment]
Haha, I just pushed #386 which has a change to the `docker run` example instead.

**Comment by brian-brazil** [comment]
:+1:

File: generator/Dockerfile [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+    go install && \
+    cp generator.yml /opt/ && \
+    make mibs && \
+    mv mibs /opt/
 
 WORKDIR "/opt"
 
 ENTRYPOINT ["/go/bin/generator"]
 
+ENV MIBDIRS mibs
```
Comment (brian-brazil) [inline generator/Dockerfile]: Actually, this values doesn't match the docs. This should work off the user's mibs, not the ones we use.

Comment (SuperQ) [reply generator/Dockerfile]: I've update the README to make things easier and match our build.

**Comment by SuperQ** [comment]
I've also removed the `make mibs` part of the Dockerfile.

What do you think about removing the `generator.yml` from the Dockerfile as well?

**Comment by brian-brazil** [comment]
It's make sense on consistency grounds.

**Comment by SuperQ** [comment]
Done, PTAL.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**